web_widget_iframe/static/src/js/web_widget_iframe.js

```diff
@@ -0,0 +1,25 @@
+odoo.define('web_widget_iframe.web_widget_iframe', function(require) {
+    "use strict";
+
+    var core = require('web.core');
+
+    var FieldChar = core.form_widget_registry.get('char');
```

Comment:
You can also use FieldUrl which maybe automatically adds the protocol etc (don't know for sure :P).